sdk/appconfiguration/Azure.Data.AppConfiguration/tests/SessionRecords/ConfigurationLiveTests/GetBatchSettingWithReadOnly.json

```diff
@@ -1,21 +1,21 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://pakrym-azconfig-ui.azconfig.io/kv/key-2113346919?label=my_label\u0026api-version=1.0",
```

Comment:
why did u have to re-record all the tests?

Reply:
There were some extra recordings so I've removed the entire directory and re-recorded.